src/components/dashboard/Rewards.web.js

```diff
@@ -23,7 +24,7 @@ const RewardsTab = props => {
     >
       <iframe
         title="Rewards"
-        src={`https://w3.gooddollar.org?loginToken=${loginToken}`}
+        src={`${Config.web3SiteUrl}?loginToken=${loginToken}`}
```

Comment:
@sirpy the correct name of query parameter for login token is just 'token' not 'loginToken'